tests/Integration/Behaviour/Features/Scenario/Order/add_cart_rule_to_order.feature

```diff
@@ -43,6 +43,14 @@ Feature: Add discounts to order from Back Office (BO)
       | taxes         | $1.85    |
       | total         | $32.65   |
 
+  Scenario: Add amount type discount to order which has no invoices with HTML characters
+    Given order "bo_order1" does not have any invoices
+    When I add discount to order "bo_order1" with following details:
+      | name      | <<           |
+      | type      | amount      |
+      | value     | 5.50         |
+    Then I should get no error
```

Comment:
```suggestion
    Then I should get no error
    And I the order "bo_order1" name is "<<"
```

Something like that

Reply:
We discussed this previously with @jolelievre . I think maybe we should enforce _everywhere_ that "I should get no error"? Like consider errors should never be thrown unless a Behat step especially mentions it?

Reply:
Fixed